### PR TITLE
Add PyCon India Tickets blog

### PR DESCRIPTION
**Pull request by vipulgupta2048**
- Added Blog for tickets sales. 
- Added a link to Convince My Boss in the same. 
- Solves #143 
**Note**
@ananyo2012 This needs to be updated constantly and we should add more details of the events as soon as it gets confirmed. 
@pythonindia/pycon-2018-team  Please review.

### CODE REVIEW

File: content/blog/pycon-india-tickets-2018.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Title: PyCon India Tickets 2018
+Date: 2018-05-31 12:45
+Category: 2018
+Slug: pycon-india-tickets-2018
+Summary: Details about the different tickets for PyCon India 2018.
+
+PyCon India 2018, the tenth edition of the premier conference in India on using and developing the Python programming language is conducted annually by the Python developer community has started ticket sales for the mega event that is scheduled from 5th - 9th October 2018. 
+
+The conference will be having a tight schedule of workshops,  keynotes, talks, workshops, and dev-sprints strung together through 3 different tracks. Any person who has a conference ticket is eligible to attend the Devsprints. The ticket structure is as follows:- **Grab the early bird tickets now from [Explara](https://in.explara.com/e/pycon-india-2018/).**
```
Comment (ananyo2012) [inline content/blog/pycon-india-tickets-2018.md]: Can we add `Grab the early bird tickets now from [Explara](https://in.explara.com/e/pycon-india-2018/).` to a separate line ?

File: content/blog/pycon-india-tickets-2018.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+### Student Ticket ###
+A student ticket reserves your seat for the conference on 6th - 7th October, 2018, with a pricing of INR 1000 (GST applicable), the sale of student tickets ends on 30 September 2018 or completion of 300 tickets. 
+
+### Regular Ticket  
+A regular ticket reserves your seat for the conference on 6th - 7th October, 2018, with a pricing of INR 1700 (GST applicable), the sale of regular tickets begins on 1st July 2018.
+
+### Late Bird Ticket ###
+A late bird ticket reserves your seat for the conference on 6th - 7th October, 2018, with a pricing of INR 2500 (GST applicable), the sale of late bird begins on 1st September 2018.
+
+### Supporter Ticket ###
```
Comment (ananyo2012) [inline content/blog/pycon-india-tickets-2018.md]: Can you remove this as it isn't been finalized yet

File: content/blog/pycon-india-tickets-2018.md (hunk 1 above)
Comment (vipulgupta2048) [reply content/blog/pycon-india-tickets-2018.md]: Sure.

File: content/blog/pycon-india-tickets-2018.md (hunk 2 above)
Comment (vipulgupta2048) [reply content/blog/pycon-india-tickets-2018.md]: I just heard talks of it. That's why.

File: content/blog/pycon-india-tickets-2018.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+Title: PyCon India Tickets 2018
+Date: 2018-05-31 12:45
+Category: 2018
+Slug: pycon-india-tickets-2018
+Summary: Details about the different tickets for PyCon India 2018.
+
+PyCon India 2018, the tenth edition of the premier conference in India on using and developing the Python programming language is conducted annually by the Python developer community has started ticket sales for the mega event that is scheduled from 5th - 9th October 2018. 
+
+The conference will be having a tight schedule of workshops,  keynotes, talks, workshops, and dev-sprints strung together through 3 different tracks. Any person who has a conference ticket is eligible to attend the Devsprints. The ticket structure is as follows:-
```
Comment (ananyo2012) [inline content/blog/pycon-india-tickets-2018.md]: For the devsprints can you mention that though the tickets will be free we will have registrations to manage the logistics ? See FAQs for more details.

Comment (vipulgupta2048) [reply content/blog/pycon-india-tickets-2018.md]: That would be better. Implementing.

**Comment by vipulgupta2048** [comment]
Changes made as requested.

**Comment by vipulgupta2048** [comment]
Don't merge this just yet.

**Comment by pushplataranjan** [comment]
@vipulgupta2048 If this PR is not complete mark it as WIP(Work in progress): (title for Pr). Easy for reviewer to skip these.

File: content/blog/pycon-india-tickets-2018.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+### Late Bird Ticket ###
+A late bird ticket reserves your seat for the conference on 6th - 7th October, 2018, with a pricing of INR 2500 (GST applicable), the sale of late bird begins on 1st September 2018.
+
+###Payment Options###
+You can pay through Citrus or EBS gateway via Credit Card, Debit Card or Netbanking. You can also use Paytm for your registration by accepting the terms and conditions of Explara.
+
+**See you there at the Conference!!**
+
+_Want to come for the conference, but not sure how to pitch the same to your boss. Leave it to us, check out [Convince My Boss](http://www.in.pycon.org/convince-my-boss.html)._
```
Comment (ananyo2012) [inline content/blog/pycon-india-tickets-2018.md]: @vipulgupta2048 Th correct link of the website is https://in.pycon.org/2018/ . Please fix this

**Comment by vipulgupta2048** [comment]
@ananyo2012 Changes made as requested. I have taken the liberty to update the blog regarding the sold-out status of early bird tickets. Please review. Ready to merge.

File: content/blog/pycon-india-tickets-2018.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+### Late Bird Ticket ###
+A late bird ticket reserves your seat for the conference on 6th - 7th October, 2018, with a pricing of INR 2500 (GST applicable), the sale of late bird begins on 1st September 2018.
+
+###Payment Options###
+You can pay through Citrus or EBS gateway via Credit Card, Debit Card or Netbanking. You can also use Paytm for your registration by accepting the terms and conditions of Explara.
+
+**See you there at the Conference!!**
+
+_Want to come for the conference, but not sure how to pitch the same to your boss. Leave it to us, check out [Convince My Boss](http://www.in.pycon.org/2018/convince-my-boss.html)._
```
Comment (ananyo2012) [inline content/blog/pycon-india-tickets-2018.md]: The link is still not correct https://in.pycon.org/2018/

**Comment by vipulgupta2048** [comment]
Implemented.

File: content/blog/pycon-india-tickets-2018.md [hunk 6]
```diff
@@ -8,18 +8,20 @@ PyCon India 2018, the tenth edition of the premier conference in India on using
 
 The conference will be having a tight schedule of workshops,  keynotes, talks, workshops, and dev-sprints strung together through 3 different tracks. Any person who has a conference ticket is eligible to attend the Devsprints. Although the Developer Sprints are free, we would be having registrations beforehand to manage the logistics. The ticket structure is as follows:-
```
Comment (realslimshanky) [inline content/blog/pycon-india-tickets-2018.md]: extra space
```workshops,  keynotes``` -> ```workshops, keynotes,```

File: content/blog/pycon-india-tickets-2018.md (hunk 5 above)
Comment (realslimshanky) [reply content/blog/pycon-india-tickets-2018.md]: `http://www.in.pycon.org/2018/convince-my-boss.html` -> `http://in.pycon.org/2018/convince-my-boss.html`

File: content/blog/pycon-india-tickets-2018.md (hunk 6 above)
Comment (ananyo2012) [reply content/blog/pycon-india-tickets-2018.md]: @vipulgupta2048 Can you directly push a commit for this fix ?

Comment (realslimshanky) [reply content/blog/pycon-india-tickets-2018.md]: Should we hotfix these things or slip in any other issue as minor fix?

Comment (vipulgupta2048) [reply content/blog/pycon-india-tickets-2018.md]: Hotfixing, I just saw this. Thanks @realslimshanky

**Pull request merged**